### DISCUSSION TITLE
Create C SDK HIP-323

### DISCUSSION THREAD

**Discussion by gregscullard** [hiero-ledger/hiero-improvement-proposals#325] [Edits]
HIP-323 proposes a new C SDK, this discussion is open to comment on the HIP (https://github.com/hashgraph/hedera-improvement-proposal/pull/323)

**Comment by mohsin-hedera** [comment]
@gregscullard shall we add that SDK is to maintain feature parity with other under-development HIPs, e.g retry mechanism, file append/contract upload, wallet-provider interface.

**Comment by gregscullard** [comment]
Added to the HIP under "Specifications"
* The C SDK must have feature-parity with other SDKs and include HIPs that are under development or to be approved
